SECURITY.md

```diff
@@ -13,18 +13,19 @@ We release updates to the Uptane specification to address security vulnerabiliti
 | 1.0.1   | :white_check_mark: |
 | 1.0.0   | :white_check_mark: |
 
-## Reporting a Vulnerability
+## Reporting an Errata
```

Comment:
Sorry to be annoying, but the singular is "erratum". Maybe it'd better to just write "Reporting errata" here?